test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -245,6 +245,7 @@ class Context {
 		case BlockType::kReturnable:
 			return "return";
 		}
+		return "";
```

Comment:
Ok, I found missing default case:).
Please, add this as the `default` case in the previous commit.

Reply:
This return was added by a mistake.